robotoff/insights/importer.py

```diff
@@ -341,6 +341,9 @@ def ignore_insight(
         category: str,
         seen_set: Set[str],
     ):
+        if product is None:
+            return False  # product not in store !
```

Comment:
I forgot to inverse the bool, we should return `True` if we ignore the insight and `False` otherwise. Can you inverse all `return True/False` and remove the `not` on L332?

Reply:
done